Source/Core/Core/State.cpp

```diff
@@ -184,10 +184,10 @@ static std::string DoState(PointerWrap& p)
 
 	PowerPC::DoState(p);
 	p.DoMarker("PowerPC");
-	HW::DoState(p);
-	p.DoMarker("HW");
 	CoreTiming::DoState(p);
```

Comment:
Can we get a comment here saying

    // CoreTiming needs to be restored before restoring Hardware because
    // the controller code might need to schedule an event if the controller has changed.